spec/factory_specs/contacts_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+describe 'Contact factory' do
+  let(:contact) { create(:contact) }
+
+  it 'creates ProviderUCASPreference object' do
+    expect(contact).to be_instance_of(Contact)
+    expect(contact).to be_valid
+  end
```

Comment:
describe 'associations' do
    it { should belong_to(:provider) }
end

??

Reply:
This is a factory spec that proves that the factory works, not a model test so no need to test associations.